### ISSUE TITLE
Unable to cancel multi-hop flight paths by logging out

### ISSUE THREAD

**Issue by jsb** [elysium-project/server#2226]
Logging out and back in again while on a taxi should cause the taxi to land at the next intermediate stop. This was a convenient way to cancel multi-hop flight paths ([source](http://wow.allakhazam.com/forum.html?forum=21&mid=1151834915229509751#4)).

This currently does not work on this server where the taxi just continues on to the final destination.

**Comment by Mostprovidedlinks** [comment]
I remember this happened on retail as well 

I agree to this Change

**Comment by towneh** [comment]
Yep, alt+f4 was indeed a working method to abort long flight times forcing your character to land at the next FP as soon as you log back in.

**Comment by Mostprovidedlinks** [comment]
I hope they fix this. Would help
